src/test/java/test/TransitionsPersonalAccountTest.java

```diff
@@ -0,0 +1,93 @@
+package test;
+
+import elements.AddInfoUser;
+import elements.Drivers;
+import elements.Users;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.html5.LocalStorage;
+import org.openqa.selenium.html5.WebStorage;
+import  pageforthetests.*;
+
+public class TransitionsPersonalAccountTest extends Drivers {
+    WebDriver driver;
+    private String url = "https://stellarburgers.nomoreparties.site/";
+    private Users user;
+    private String accessToken;
+    BasePage basePage;
+    AutorizationPage autorizationPage;
+    RegistrationPage registrationPage;
+    PersonalAccountPage personalAccountPage;
+
+
+    @Before
+    public void settingUp() {
+        Drivers browserPage = new Drivers();
+        driver = browserPage.getWebdriver("chrome");
+        driver.get(url);
+        user = Users.randomDataUser();
+        AddInfoUser.buildUser(user);
+        basePage = new BasePage(driver);
+        autorizationPage = new AutorizationPage(driver);
+        registrationPage = new RegistrationPage(driver);
+        personalAccountPage = new PersonalAccountPage(driver);
+    }
+    @Test
+
+    public void testGoToAccountPageGoToBurger() {
+        basePage.clickLoginButton();
+        autorizationPage.waitLoginPage();
+        autorizationPage.clickLogoButton();
+        basePage.waitForLoadMainPage();
+        Assert.assertTrue(driver.findElement(basePage.makeBurger).isDisplayed());
+    }
+
+    @Test
+
+    public void testExitPersonalAccountPage() {
+        basePage.clickAccountButton();
+        autorizationPage.waitLoginPage();
+        autorizationPage.logInUser(user.getEmail(), user.getPassword());
+        LocalStorage localStorage = ((WebStorage) driver).getLocalStorage();
+        accessToken = localStorage.getItem("accessToken");
+        basePage.waitForLoadMainPage();
+        basePage.clickAccountButton();
```

Comment:
Здесь тест падает с ошибкой:

`org.openqa.selenium.ElementClickInterceptedException: Element <a class="AppHeader_header__link__3D_hX" href="/account"> is not clickable at point (1176,44) because another element <div class="Modal_modal_overlay__x2ZCr"> obscures it
`

Reply:
у меня это тест проходит успешно
![image](https://github.com/user-attachments/assets/d5ae7faa-78d6-43f8-9d9b-1c2e6c3560fb)
![image](https://github.com/user-attachments/assets/e9118a1a-10f5-4bf5-9e6f-27ed704471b1)